src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -245,6 +245,7 @@ public static void GetPathRoot_Unix()
         [InlineData(@"[:\\")]       // 091 = [     090 = Z
         [InlineData(@"`:\foo")]    // 096 = `     097 = a
         [InlineData(@"{:\\")]       // 123 = {     122 = z
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Bug fixed on Core where it would return true if the first char is not a drive letter followed by a VolumeSeperatorChar coreclr/10297")]
```

Comment:
Just tested this out in a ConsoleApp targeting 462 and it returns true. Was this fixed on 47?